docs/querying/api.md

```diff
@@ -1022,3 +1022,205 @@ $ curl -XPOST http://localhost:9090/api/v1/admin/tsdb/clean_tombstones
 ```
 
 *New in v2.1 and supports PUT from v2.9*
```

Comment:
yeah that's normal, this line belongs to the tsdb endpoints ;). So normally it should be ok.
Perhaps I should add a line like this one. But no idea in which version it will be added.